GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+This document is aimed at getting started with github if you never used it.
+
+If you're a developer there's more info at [README.md](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/master/README.md).
```

Comment:
If you could adjust this to be a link to the relative file.  I think if this is in docs, it would just link to `../Readme.md`